splunk_otel/options.py

```diff
@@ -118,6 +119,7 @@ def _get_resource(
         attributes = attributes or {}
         if service_name:
             attributes[_SERVICE_NAME_ATTR] = service_name
+            attributes[_SPLUNK_SERVICE_NAME_ATTR] = service_name
```

Comment:
Does our backend not read the "service.name" attribute? I thought it did.

Reply:
I guess this is a fallback way of generating a service.name in case it's not provided?

Reply:
It is.

Reply:
Sorry, can you help me understand? How is this a way to generate a service name in case it's not provided?

Reply:
Apologies for the confusion. This is a way to set the `sf_service` service name tag if the collector is not doing the conversion. Both are set from the service name. The default service name is being set globally to `unnamed-python-service` if it is not set by the user.

Reply:
Got it, thank you. Seems pretty benign, but maybe unnecessary for production? I'm under the impression that `service.name` is all you need -- and that the splunk otel collector doesn't set `sf_service` anyway (not seeing it in the codebase).